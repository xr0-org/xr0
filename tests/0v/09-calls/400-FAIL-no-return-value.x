int
f()
{
	int k;

	return k;
}
