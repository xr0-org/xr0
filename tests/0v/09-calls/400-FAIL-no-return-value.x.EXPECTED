09-calls/400-FAIL-no-return-value.x:6:10: returned expression has no value (f)
